src/test/eventlisteners.ts

```diff
@@ -38,25 +40,25 @@ describe('event listeners', function() {
     assert.deepEqual(result, [1, 2]);
   });
   it('does calls handler for function in array', function() {
-    var result = [];
-    function clicked(ev) { result.push(ev); }
-    var vnode = h('div', {on: {click: [clicked, 1]}}, [
+    var result: Event[] = [];
+    function clicked(ev: Event) { result.push(ev); }
+    var vnode = h('div', {on: {click: [clicked, 1] as any}}, [
       h('a', 'Click my parent'),
     ]);
     elm = patch(vnode0, vnode).elm;
     elm.click();
     assert.deepEqual(result, [1]);
   });
   it('handles changed value in array', function() {
-    var result = [];
-    function clicked(ev) { result.push(ev); }
-    var vnode1 = h('div', {on: {click: [clicked, 1]}}, [
+    var result: Event[] = [];
+    function clicked(ev: Event) { result.push(ev); }
+    var vnode1 = h('div', {on: {click: [clicked, 1] as any}}, [
       h('a', 'Click my parent'),
     ]);
-    var vnode2 = h('div', {on: {click: [clicked, 2]}}, [
+    var vnode2 = h('div', {on: {click: [clicked, 2] as any}}, [
       h('a', 'Click my parent'),
     ]);
-    var vnode3 = h('div', {on: {click: [clicked, 3]}}, [
+    var vnode3 = h('div', {on: {click: [clicked, 3] as any}}, [
```

Comment:
It seems that due to #431 type checking must be disabled here. And a few similar cases below.